plugins/storage/pyparts/ceph_daemon_checks.py

```diff
@@ -179,10 +188,23 @@ def get_ceph_versions_mismatch(self):
             for version in versions[daemon_type]:
                 vers.append(version)
                 global_vers.add(version)
-
+                if self.version_compare(str(version)) > \
```

Comment:
if version is not already a string we should fix that in https://github.com/canonical/hotsos/blob/master/core/plugins/storage/ceph.py#L134 rather than having to convert on the fly.

Reply:
I verified the conversion to string is not necessary, I will remove that